pdf-rendering-template/bin/render-pdf.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * node.js script to render a given HTML file into the given PDF filename.
+ * Details of the API are at https://developers.google.com/web/tools/puppeteer/
+ *
+ * This command is invoked as:
+ *
+ *   $ node bin/render-pdf.js input.html output.pdf
+ *
+ */
+
+const puppeteer = require("puppeteer");
+
+const inputPath = `file://${process.argv[2]}`;
+const outputPath = process.argv[3];
+
+console.log("Input path:", inputPath);
+console.log("Output path:", outputPath);
+
+(async () => {
+  const browser = await puppeteer.launch();
```

Comment:
It would be good if we could take an env value here for `executablePath`: `{ executablePath: process.env.ACKAMA_PATH_TO_CHROME || undefined }` - that'll let folks on WSL run this, as we need to point puppeteer explicitly to our Chrome.

This will be fine on other platforms as passing in `undefined` will trigger the default "find chrome yourself" behaviour of puppeteer.

Reply:
Have added this option now